examples/microprofile-config/microprofile-config-app-s2i.yaml

```diff
@@ -1,7 +1,7 @@
 build:
   mode: s2i
   uri: https://github.com/wildfly/quickstart.git
-  ref: 24.0.0.SP1
+  ref: 24.0.1.Final
```

Comment:
I have my doubts here, quickstarts are related to the server image, my bet was if we were using the latest server image, this time based on WF 24.0.1, we should use the quickstarts based on that server release, hence QS for WF 24.0.1.

@jmesnil Is there no correlation between the QS release and the base server release?

Reply:
@yersan you were right to update the examples to use the `24.0.1.Final` tag. 

What I mean is that updating the chart examples does not require a new release of the Helm Chart itself.

Reply:
ahhh, alright, now I see it :), since the `mayor.minor` already includes the latest patches, the only thing we should have done was to bump the QS tag, that was all needed. Now I got it.